scripts.py

```diff
@@ -29,12 +29,12 @@ def check_mypy():
 
 
 def test():
-    subprocess.run(["python", "-m", "pytest", "--log-level=CRITICAL"], check=True)
+    subprocess.run(["python", "-m", "pytest", "-n", "6", "--log-level=CRITICAL"], check=True)
 
 
 def test_verbose():
     subprocess.run(
-        ["python", "-m", "pytest", "-vv", "-s", "--log-level=CRITICAL"], check=True
+        ["python", "-m", "pytest", "-n", "6", "-vv", "-s", "--log-level=CRITICAL"], check=True
```

Comment:
I'd use "auto" here. As a data point, if I use "auto" and set Docker's resources to use all cores, pytest assigns 14 cores and I see a ~16s speedup locally (76.39s with 6 cores, 61.40s with "auto"/14 cores).

Reply:
Nice!